pkgs/development/python-modules/libtmux/default.nix

```diff
@@ -32,12 +33,16 @@ buildPythonPackage rec {
   nativeCheckInputs = [
     procps
     tmux
+    ncurses
     pytest-rerunfailures
     pytestCheckHook
   ];
 
   pytestFlagsArray = [
     "tests"
+
+    # Fail with: 'no server running on /tmp/tmux-1000/libtmux_test8sorutj1'.
+    "-k 'not test_new_session_width_height'"
```

Comment:
Consider using `disabledTests` and `disabledTestPaths` from pytestCheckHook, it's cleaner:
https://nixos.org/manual/nixpkgs/stable/#python

Reply:
Fixed. I cannot test Darwin so let us wait the CI to complete.